Mathmagician/Program.cs

```diff
@@ -59,8 +66,15 @@ static void Main(string[] args)
                 isItAnInt = (Int32.TryParse(makeIntoAnInt, out printThisManyNumbers));
                 if (isItAnInt)
                 {
-                    Console.WriteLine("Printing " + printThisManyNumbers + " Fibonaccis! MAGIC!");
-                    //call number-generating method here. ex: Integers.GenerateIntegers(printThisManyNumbers)
+                    if (printThisManyNumbers < 0)
+                    {
+                        Console.WriteLine("A negative? Sounds like you owe me some numbers! Pay up! Or, press a key to exit.");
```

Comment:
@ryan-b-writin Consider refactoring this so that you only need to write like logic once.